compose/musicbrainz-dev.yml

```diff
@@ -17,7 +17,7 @@ services:
     depends_on:
       - validator
   validator:
-    image: validator/validator:20.3.16
+    image: ghcr.io/validator/validator
```

Comment:
I see that there is also a 21.7.0 release on ghcr: https://github.com/validator/validator/pkgs/container/validator/versions. Is there a specific reason that you used :latest instead of this? It might be a good idea to stay with a specific version to avoid errors in the future if something changes

Reply:
@alastair Thanks for the review. I agree that a specific version would be better. I will update the PR to the proposed release then :)